src/backend/vulkan/src/lib.rs

```diff
@@ -557,6 +557,11 @@ impl hal::PhysicalDevice<Backend> for PhysicalDevice {
             && self.properties.device_id & info::intel::DEVICE_KABY_LAKE_MASK
                 == info::intel::DEVICE_KABY_LAKE_MASK;
 
+        let is_windows_intel_sky = cfg!(windows)
```

Comment:
let's have a single boolean for this workaround, e.g. `is_windows_intel_dual_src_bug`